### ISSUE TITLE
Nothing wrong. Just saying thanks. 

### ISSUE THREAD

**Issue by tonydiep** [vfsfitvnm/ViMusic#993]
### Steps to reproduce the bug

Nothing's wrong. Just discovered this app and am grateful you made it. 

### Expected behavior

NA

### Actual behavior

NA

### ViMusic version

NA

### Android version

NA

**Comment by dAtA-TRoN** [comment]
I agree. Thanks Mr. Dev, I use the app every day. It's my favorite app along with "NewPipe". Very well done 🫶🏼!

**Comment by arijit4** [comment]
I'm hoping this app again gets into maintainance
